app/controllers/api/v2/moves_filtered_actions.rb

```diff
@@ -0,0 +1,5 @@
+module Api::V2
+  module MovesFilteredActions
+    include Api::V2::MovesActions
```

Comment:
Are all the methods in `Api::V2::MovesActions` required? I think only the `index_and_render` method is necessary? its probably not wise to include `create_and_render`, `update_and_render` etc